prime-router/src/test/kotlin/FakeReportTests.kt

```diff
@@ -195,42 +195,4 @@ class FakeReportTests {
         val expected = "Any lab USA, Any facility USA"
         assertThat(actual).isEqualTo(expected)
     }
-
-    @Test
```

Comment:
Context: Tests that require the full metadata catalog were moved to be integration tests